layouts/default.vue

```diff
@@ -546,7 +559,14 @@ export default {
               :show-header="!g.isRoot"
               @selected="groupSelected($event)"
               @expand="groupSelected($event)"
-            />
+            >
+              <template #header>
+                <h6 class="root-group">
+                  <i v-if="g.icon" class="icon" :class="g.iconClass" />
+                  <div>{{ g.label }}</div>
```

Comment:
I don't think `div` is technically valid content for `h6`, perhaps `span` would be more appropriate here?

https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements